### PR TITLE
Pixel Density And Scale

### LINKED ISSUE #107: [SDL 0179] Pixel density and Scale
Proposal: [Pixel density and Scale](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0179-pixel-density-and-scale.md)

> This proposal is about adding parameters to `VideoStreamingCapabilities` for pixel density and scale.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/526

### PR DESCRIPTION
**Pull request by Jack-Byrne**
Fixes #107

### CODE REVIEW

**Review by theresalech — APPROVED** [review]

**Review by theresalech — CHANGES_REQUESTED** [review]
This PR should be targeting `version/6_0_0 `

File: MOBILE_API.xml [hunk 1]
```diff
@@ -2872,6 +2872,15 @@
         <param name="hapticSpatialDataSupported" type="Boolean" mandatory="false">
             <description>True if the system can utilize the haptic spatial data from the source being streamed. If not included, it can be assumed the module doesn't support haptic spatial data'. </description>
         </param>
+        <param name="diagonalScreenSize" type="Float" mandatory="false" since="6.0">
+            <description>The diagonal screen size in inches.</description>
+        </param>
+        <param name="pixelPerInch" type="Float" mandatory="false" since="6.0">
+            <description>PPI is the diagonal resolution in pixels divided by the diagonal screen size in inches.</description>
+        </param>
+        <param name="scale" type="Float" mandatory="false" since="6.0">
```
Comment (iCollin) [inline MOBILE_API.xml]:
```suggestion
        <param name="scale" type="Float" minvalue="1" maxvalue="10" mandatory="false" since="6.0">
```

**Comment by Jack-Byrne** [comment]
@theresalech @iCollin Updated

File: MOBILE_API.xml [hunk 2]
```diff
@@ -2872,6 +2872,15 @@
         <param name="hapticSpatialDataSupported" type="Boolean" mandatory="false">
             <description>True if the system can utilize the haptic spatial data from the source being streamed. If not included, it can be assumed the module doesn't support haptic spatial data'. </description>
         </param>
+        <param name="diagonalScreenSize" type="Float" mandatory="false" since="6.0">
```
Comment (theresalech) [inline MOBILE_API.xml]: Need `minvalue` of `1` per [proposal revisions](https://github.com/smartdevicelink/sdl_evolution/issues/808)

File: MOBILE_API.xml [hunk 3]
```diff
@@ -2872,6 +2872,15 @@
...
+            <description>The diagonal screen size in inches.</description>
+        </param>
+        <param name="pixelPerInch" type="Float" mandatory="false" since="6.0">
```
Comment (theresalech) [inline MOBILE_API.xml]: Need `minvalue` of `1` per [proposal revisions](https://github.com/smartdevicelink/sdl_evolution/issues/808)

**Review by theresalech — CHANGES_REQUESTED** [review]
@JackLivio please let me know if you have any questions about my review, thanks!

File: MOBILE_API.xml (hunk 3 above)
Comment (Jack-Byrne) [reply MOBILE_API.xml]: I believe the revisions had  minvalue="0" for pixelPerInch and diagonalScreenSize

**Comment by Jack-Byrne** [comment]
@theresalech Updated

**Review by theresalech — APPROVED** [review]

**Review by joeygrover — APPROVED** [review]

**Pull request merged**